src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -715,6 +715,8 @@ private void SearchPackagesAndRefreshUpdateCount(bool useCacheForUpdates)
                     useCacheForUpdates: useCacheForUpdates,
                     pSearchCallback: null,
                     searchTask: null);
+
+                IsTabChanged = false;
```

Comment:
Would this be better suited for the end of `Filter_SelectionChanged` ?

Reply:
Removed. Not needed for this PR